src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -1270,6 +1273,12 @@ public static void modifyAttributes()
         });
     }
 
+    public static void onBlockEntityBlocks(BiConsumer<BlockEntityType<?>, Block> callback)
+    {
+        final var event = new AddValidBlocksToBlockEntityEvent(callback);
+        ModLoader.get().postEvent(event);
```

Comment:
Could simplify this by inlining the event into the `postEvent` call.
```suggestion
        ModLoader.get().postEvent(new AddValidBlocksToBlockEntityEvent(callback));
```